packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -25,98 +32,74 @@ describe('authentication', () => {
   });
 
   it('should not allow authentication.authenticate calls before initialization', () => {
+    expect.assertions(1);
     const authenticationParams: authentication.AuthenticatePopUpParameters = {
       url: 'https://someurl/',
       width: 100,
       height: 200,
     };
 
-    expect(() => authentication.authenticate(authenticationParams)).toThrowError(
+    return expect(() => authentication.authenticate(authenticationParams)).toThrowError(
```

Comment:
I'd prefer that we use async/await rather than adding a return here for this test.

Reply:
We don't need either here since `() => authentication.authenticate()` is not an async function. Reverted these to how they were before.